src/types/deno.d.ts

```diff
@@ -0,0 +1,8 @@
+declare namespace Deno {
+  function writeTextFile(path: string, data: string): Promise<void>;
+  function readTextFile(path: string): Promise<string>;
+  function mkdir(
+    path: string,
+    options?: { recursive?: boolean }
+  ): Promise<void>;
+}
```

Comment:
Do we need to declare a type file like this?

Reply:
Try adding `deno.ns` to the tsconfig in the relevant fields

https://docs.deno.com/runtime/reference/ts_config_migration/#using-the-%22lib%22-property

```
{
  "compilerOptions": {
    "lib": ["deno.ns"]
  }
}
```

Use the Deno target with the smallest possible scope